[2024-02-29T14:07:37.354+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dataIngestionDag.push_csv_to_s3 manual__2024-02-29T14:07:19.647487+00:00 [queued]>
[2024-02-29T14:07:37.360+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dataIngestionDag.push_csv_to_s3 manual__2024-02-29T14:07:19.647487+00:00 [queued]>
[2024-02-29T14:07:37.360+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 3
[2024-02-29T14:07:37.375+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): push_csv_to_s3> on 2024-02-29 14:07:19.647487+00:00
[2024-02-29T14:07:37.380+0000] {standard_task_runner.py:60} INFO - Started process 8156 to run task
[2024-02-29T14:07:37.384+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dataIngestionDag', 'push_csv_to_s3', 'manual__2024-02-29T14:07:19.647487+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/ingestionPipeline.py', '--cfg-path', '/tmp/tmpsfwtnpc6']
[2024-02-29T14:07:37.386+0000] {standard_task_runner.py:88} INFO - Job 110: Subtask push_csv_to_s3
[2024-02-29T14:07:37.423+0000] {task_command.py:423} INFO - Running <TaskInstance: dataIngestionDag.push_csv_to_s3 manual__2024-02-29T14:07:19.647487+00:00 [running]> on host ip-172-31-12-93.ap-south-1.compute.internal
[2024-02-29T14:07:37.490+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='shubh.sehgal2506@gmail.com' AIRFLOW_CTX_DAG_OWNER='shubh-airflow' AIRFLOW_CTX_DAG_ID='dataIngestionDag' AIRFLOW_CTX_TASK_ID='push_csv_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T14:07:19.647487+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-29T14:07:19.647487+00:00'
[2024-02-29T14:07:37.769+0000] {python.py:202} INFO - Done. Returned value was: CSV FILE PUSHED TO S3
[2024-02-29T14:07:37.784+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=dataIngestionDag, task_id=push_csv_to_s3, execution_date=20240229T140719, start_date=20240229T140737, end_date=20240229T140737
[2024-02-29T14:07:37.836+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-29T14:07:37.849+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
